apps/browser/src/vault/fido2/content/content-script.ts

```diff
@@ -61,12 +61,26 @@ async function isLocationBitwardenVault(activeUserSettings: Record<string, any>)
   return window.location.origin === activeUserSettings.serverConfig.environment.vault;
 }
 
-function initializeFido2ContentScript() {
-  const s = document.createElement("script");
-  s.src = chrome.runtime.getURL("content/fido2/page-script.js");
-  (document.head || document.documentElement).appendChild(s);
+const messenger = Messenger.forDOMCommunication(window);
+
+function injectPageScript() {
+  // Locate an existing page-script on the page
+  const existingPageScript = document.getElementById("bw-fido2-page-script");
+
+  // Inject the page-script if it doesn't exist
+  if (!existingPageScript) {
+    const s = document.createElement("script");
+    s.src = chrome.runtime.getURL("content/fido2/page-script.js");
+    s.id = "bw-fido2-page-script";
+    (document.head || document.documentElement).appendChild(s);
+  } else {
+    // If the page-script already exists, send a reconnect message to the page-script
+    messenger.sendReconnectCommand();
+  }
```

Comment:
```suggestion
  if (!existingPageScript) {
    const s = document.createElement("script");
    s.src = chrome.runtime.getURL("content/fido2/page-script.js");
    s.id = "bw-fido2-page-script";
    (document.head || document.documentElement).appendChild(s);
    
    return;
  }
  
  // If the page-script already exists, send a reconnect message to the page-script
  messenger.sendReconnectCommand();
```
Nit, changes not required - You could consider doing an early return here to clean up the logic flow a bit.